OneCommunityGlobal/HighestGoodNetworkApp
# Roberto- fixes intangible time entries round to 2nd decimal place

# Description
The intangible time entry time in a user profile is not rounded to 2 decimals.
<img width="814" alt="Screenshot 2023-11-21 at 4 32 45 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/117053202/a5662e2d-7add-4511-a82c-3324626a5d11">


## Related PRS (if any):
Use latest backend dev 

## Main changes explained:
- Added logic to VolunteerTimesTab.jsx to prohibit user from entering value into input with more than 2 decimal points.
- To TimeEntryForm.jsx, UserProfile.jsx, and DeleteModal.jsx added basic logic to round the intangible time.
- Removed userProfile value from props in DeleteModal.jsx and retrieved that value from redux state, fixes a deleting time entry problem not updating the hours correctly.

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as any user
4. go to your profile page
5. verify that the intangible time if any is rounded properly.
6. go to timelog → add some intangible time → go back to your profile
7. verify that the changes resulted in rounded time accordingly.
8. verify that removing intangible time entries works as well.

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/117053202/f448395c-96b7-4e5d-93ec-969bf6c69402



## Note:
Switching the tangibility of existing time might result in the intangible hours being affected poorly. - Looks like another developer is working on a pr for this.
